### PR TITLE
Added redirect page key while duplicating form

### PR DESCRIPTION
**Pull request by rohilmistry**
Not tested with latest version.

### CODE REVIEW

File: cf7-success-page-redirects.php [hunk 1]
```diff
@@ -84,6 +84,21 @@ function cf7_success_page_save_contact_form( $contact_form ) {
...
+ */
+
+function cf7_after_form_create($my_form){
+
+	//Get old form ID
+	if(isset($_REQUEST['post']) || $_REQUEST['post'] != ''){
+		$old_id	=	get_post_meta($_REQUEST['post'], '_cf7_success_page_key', true);
+	}
+	//Update new duplicate form
+	update_post_meta( $my_form->id, '_cf7_success_page_key', $old_id );
```
Comment (rnevius) [inline cf7-success-page-redirects.php:98]: Using `$my_form->id` throws a warning. We should be using `id()` now.

File: cf7-success-page-redirects.php [hunk 2]
```diff
@@ -84,6 +84,21 @@ function cf7_success_page_save_contact_form( $contact_form ) {
...
+}
+
+add_action( 'wpcf7_after_create', cf7_after_form_create );
```
Comment (rnevius) [inline cf7-success-page-redirects.php:102]: `cf7_after_form_create` should be a string... `'cf7_after_form_create'`

**Comment by rnevius** [comment]
@rohilmistry , I added some comments to your last commit. If you can fix those small issues, I'll merge the request.

**Comment by rohilmistry** [comment]
Done !

**Pull request merged**